sources/Context.h

```diff
@@ -53,8 +52,9 @@ class Context {
    * @brief Constructor
    *
    * @param def The context definition
+   * @param config simulation configuration
```

Comment:
it is not simulation configuration rather than DFL specific configuration

Reply:
done